### ISSUE TITLE
FreeBSD: System.Console is not working right

### ISSUE THREAD

**Issue by wfurt** [dotnet/corefx#24259]
This is because terminfo database is missing in default installation. 
I tried to add ncurses package from ports. 
That only creates /usr/local/share/misc/terminfo.db
And the terminal information is kept in hashed form.

We shall investigate.

**Comment by wfurt** [comment]
this is part of https://github.com/dotnet/corefx/issues/1626

**Comment by RussellHaley** [comment]
https://forums.freebsd.org/threads/8101/

Does this info help? does termcap(5) provide what you need? If not, there is a suggestion for getting the devel/ncurses package.

**Comment by am11** [comment]
NetBSD seems to have issues with `terminfo` as well: #8032. Maybe there is a common fix for console on BSDs?

**Comment by wfurt** [comment]
yes I saw that. That means manual rebuild of ports. We can possibly give up on fancy parts - like colors - for now. I think termcap is equivalent information but somebody would need to write code to deal with it.

For now, I just copy terminfo from Linux to get by.

**Comment by wfurt** [comment]
I did not try recently but I think so as everything is collapsed in single  /usr/local/share/misc/terminfo.db file
